### DISCUSSION TITLE
[@types/npmcli__arborist] has incorrect options type for `arborist.loadActual(options)`

### DISCUSSION THREAD

**Discussion by ghiscoding** [DefinitelyTyped/DefinitelyTyped#70038] [Issues with a @types package]
I've opened this earlier in the NPM repo via this [issue](https://github.com/npm/npm-packlist/issues/238) 

### Current Behavior

The DefinitelyTyped `npmcli__arborist` is not matching the actual options from `arborist.loadActual(options)`

https://github.com/npm/cli/blob/6f33d74f310fa27aad30fd00d58d8e4404ef8cb2/workspaces/arborist/lib/arborist/load-actual.js#L98-L107

it seems that none of the options matches what is available from DefinitelyTyped `npmcli__arborist` referenced below
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3c78f0fd3c1bd7c16bde2b58e3dccadfccbdb0f1/types/npmcli__arborist/index.d.ts#L31-L79

### Expected Behavior

DefinitelyTyped `npmcli__arborist` should match these options but it does not

https://github.com/npm/cli/blob/6f33d74f310fa27aad30fd00d58d8e4404ef8cb2/workspaces/arborist/lib/arborist/load-actual.js#L98-L107

[a comment by an automated account is not shown]

**Comment by forivall** [comment]
The definitely typed typings are for arborist v5, but the current version you've linked to is v10. The types simply need to be updated, and you can contribute to improve that.

Also, the global flag is there.
